pkg/compiler/backend/simpledocker.go

```diff
@@ -47,21 +47,49 @@ func (*SimpleDocker) BuildImage(opts compiler.CompilerBackendOptions) error {
 	path := opts.SourcePath
 	dockerfileName := opts.DockerFileName
 	context := opts.Context
+	writer := NewBackendWriter(!config.LuetCfg.GetGeneral().ShowBuildOutput)
```

Comment:
suggestion: better group the common code as a function in `common.go`.  E.g. a `fn` that takes `cmd` as input, and sets it as it should, without having to declare the writer here.

Reply:
Easier also then to unit-test it

Reply:
yeah, i thought to create a wrapper for this.